frontend/src/tabs/authorship.pug

```diff
@@ -37,18 +37,26 @@
       .fileType
         input.radio-button--checkbox(type="radio", value="checkboxes", v-model="filterType")
         .checkboxes.mui-form--inline(v-if="files.length > 0")
-          label
+          label(
```

Comment:
minor style nit: maybe make it oneline here? since there's only one attribute

There's an extra space before the `:` in `color : #ffffff` too